packages/block-library/src/index.js

```diff
@@ -120,7 +121,10 @@ export const registerCoreBlocks = () => {
 		if ( ! block ) {
 			return;
 		}
-		const { name, settings } = block;
+		const { metadata, settings, name = metadata.name } = block;
+		if ( metadata ) {
+			unstable__setBlockMetadata( metadata ); // eslint-disable-line camelcase
```

Comment:
Can't we have just a single API instead of having both `unstable__bootstrapServerSideBlockDefinitions` and `unstable__setBlockMetadata`?

Reply:
Yes, I think we can consolidate under one. I wanted to have something working real quick :)

Reply:
Updated `unstable__bootstrapServerSideBlockDefinitions` to support appending as well.